source/INIT.sqf

```diff
@@ -170,14 +170,22 @@ if (isMultiplayer) then {
 	_revive_activated = paramsArray select 0; // Revives, true or false
 	DUWSMP_CP_death_cost = paramsArray select 1;
 
-	if (support_armory_available) then {
-        hq_blu1 addaction ["<t color='#ff1111'>Armory</t>","VAS\open.sqf", "", 0, true, true, "", "_this == player"];
-	};
+        if (support_armory_available) then {
+            hq_blu1 addaction ["<t color='#ff0066'>Armory 1 (VAS)</t>","VAS\open.sqf", "", 0, true, true, "", "_this == player"];
+            hq_blu1 addaction ["<t color='#ff0066'>Armory 2 (VA)</t>","bisArsenal.sqf", "", 0, true, true, "", "_this == player"];
+            { _x addaction ["<t color='#ff0066'>Armory 1 (VAS)</t>","VAS\open.sqf", "", 0, true, true, "", "_this == player"];
+              _x addaction ["<t color='#ff0066'>Armory 2 (VA)</t>","bisArsenal.sqf", "", 0, true, true, "", "_this == player"]; } forEach (Array_of_FOBS);
```

Comment:
I'm not asking you to fix this for this PR but I have been trying to make the consistency for statements similar to this...
```
if (support_armory_available) then {
    hq_blu1 addaction ["<t color='#ff0066'>Armory 1 (VAS)</t>","VAS\open.sqf", "", 0, true, true, "", "_this == player"];
    hq_blu1 addaction ["<t color='#ff0066'>Armory 2 (VA)</t>","bisArsenal.sqf", "", 0, true, true, "", "_this == player"];
    {
        _x addaction ["<t color='#ff0066'>Armory 1 (VAS)</t>","VAS\open.sqf", "", 0, true, true, "", "_this == player"];
        _x addaction ["<t color='#ff0066'>Armory 2 (VA)</t>","bisArsenal.sqf", "", 0, true, true, "", "_this == player"]; 
    } forEach (Array_of_FOBS);
};
```
I just feel at a quick glace that it is easier to notice a loop is being done here than with the version in this PR. Again, I'm not asking you to change it right now but I did want to mention it as I will most likely change it the next time I touch this file.